lib/hooks/queries/useInfiniteMarkets.ts

```diff
@@ -86,12 +86,13 @@ export const useInfiniteMarkets = (
         status_in: statuses.length === 0 ? undefined : statuses,
         tags_containsAny: tags.length === 0 ? undefined : tags,
         pool_isNull: withLiquidityOnly ? false : undefined,
-        pool:
-          currencies.length === 0
-            ? undefined
-            : {
-                baseAsset_in: currencies,
-              },
+        pool: withLiquidityOnly
```

Comment:
I dont belive this is correct since you are either just filtering for pools that have liquidity or for currencies. Although filtering by ztgQuantity wouldnt make sense if the base asset is like aUSD, but I guess ztgQuantity will have to be renamed to baseAssetQuantity.

So I actually think you can leave it like this for now since we only have one base asset.